src/editpage/components/assistance/AssistanceBar.vue

```diff
@@ -37,6 +37,11 @@ export default {
   props: {
     eventBus: Object,
   },
+  computed: {
+    hasQuery: function() {
+      return this.searchQuery !== '';
```

Comment:
```suggestion
      return this.searchQuery.trim() !== '';
```

Maybe nice?

Reply:
Searching just for spaces doesn't make much sense.
However searching for `"f"` and `"f "` gives different results (as well as `" f"`). And although that might be confusing for the user I don't know the use case so don't want to take that away.

Reply:
very good catch!